src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -14,7 +14,7 @@
 COMMENT_PARTS = {
     ClangTidyIssue: {
         'defect': ' - {nb} found by clang-tidy',
-        'analyzer': ' - `./mach static-analysis check path/to/file.cpp` (C/C++)',
+        'analyzer_custom': ' - `./mach static-analysis check {files}` (C/C++)',
```

Comment:
You can keep using the `analyzer` key here, and always provide the `files` during the formatting. That will work in both cases (when `files` is required, and when `files` is missing)

Example:

```python
>>> x = 'Test {x}'
>>> y = 'Test noX'
>>> x.format(x='123')
'Test 123'
>>> y.format(x='123')
'Test noX'
```

Reply:
Thanks for this!